src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -280,6 +280,13 @@ async def respond_to_invalid_orders(
             )
             await self.send_message(sender, str(message), messages_data)
 
+    async def log_stance_change(self, stance_log) -> None:
+        for pw in list(self.game.get_map_power_names()):
+            if pw == self.power_name:
+                continue
+            log_data = self.game.new_log_data(body=stance_log[self.power_name][pw])
```

Comment:
This kind of loop is common enough that I created a field to store a list of all other powers:

```suggestion
        for pw in self.opponents:
            log_data = self.game.new_log_data(body=stance_log[self.power_name][pw])
```